app/commands/v2/publish.rb

```diff
@@ -171,19 +178,35 @@ def update_dependencies?
         EditionDiff.new(edition).field_diff.present?
       end
 
-      def send_downstream(content_id, locale, update_type)
-        return unless downstream
-
+      def send_downstream_live
         queue = update_type == 'republish' ? DownstreamLiveWorker::LOW_QUEUE : DownstreamLiveWorker::HIGH_QUEUE
-
         DownstreamLiveWorker.perform_async_in_queue(
           queue,
+          live_worker_params.merge(worker_params)
```

Comment:
I feel like this merge belongs in the live_worker_params method, what do you think?